<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JuliaDialog</class>
 <widget class="QDialog" name="JuliaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Julia set</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="iterationsLabel">
         <property name="text">
          <string>Iterations:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="iterationsSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Scale:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="scale">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="complexLabel">
         <property name="text">
          <string>Complex parameter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="realSpin">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="plus">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="complexSpin">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="i">
         <property name="text">
          <string>i</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>JuliaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>JuliaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
